exonum/src/node/consensus.rs

```diff
@@ -617,6 +617,12 @@ impl NodeHandler {
         let mut fork = self.blockchain.fork();
         {
             let mut schema = Schema::new(&mut fork);
+            if schema.transactions().contains(&hash) && !schema.transactions_pool().contains(&hash)
```

Comment:
Shouldn't we change [that line](https://github.com/exonum/exonum/blob/master/exonum/src/node/consensus.rs#L554)?